kivy/__init__.py

```diff
@@ -78,6 +79,49 @@
         '1.11.0 - the last Kivy release that still supports Python 2.')
 
 
+###############################################################################
```

Comment:
Can you please remove the following comments and instead post them to the pull request. Then, you can simply link to the PR in a single line comment.

As it is, the comments are too in depth and to be included in kivy because having them in the issue should be enough.